JavaScript/Timer.js

```diff
@@ -0,0 +1,63 @@
+//this code is written to operationalize a timer with 3 main functions of Start, Stop and Reset.
+class Clock {
+    /*fourth take */
+
+    constructor(displayElement) {
+        this.display = displayElement;
+        this.state = "initial";
+        this.seconds = 0;
+        this.interval = null;
+    }
+
+    start() {
+        if (this.state === "initial") {
+            this.interval = setInterval(() => this.timer(), 1000);
+            this.state = "running"
+        }
+    }
+
+    stop() {
+        if (this.state ==="running"){
+        clearInterval(this.interval);
+        this.state = "stopped";
+        }
+    }
+
+    reset() {
+        this.state = "initial";
+        clearInterval(this.interval);
+        this.seconds = 0;
+        this.display.innerText = "00:00:00";
+    }
+
+    timer() {
+
+
+        let hrs = Math.floor(this.seconds / 3600);
+        let min = Math.floor((this.seconds - hrs * 3600) / 60);
+        let sec = this.seconds % 60;
+
+        if (sec < 10) sec = "0" + sec;
+        if (min < 10) min = "0" + min;
+        if (hrs < 10) hrs = "0" + hrs;
```

Comment:
You can use `padStart` instead. Also, to remove duplication, use a function here.

Reply:
made a function with the name padWithZero and used the `padStart( ) ` method in it.